frontend/src/components/home/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Sidebar from '../sidebar';
+import homeStyles from './home.module.scss';
+
+const Home = () => (
+  <>
+    <div className={homeStyles.sidebar}>
+      <Sidebar />
+    </div>
+    <div />
+  </>
+);
```

Comment:
When creating React components I would be inclined to write them a little differently. First, delcare the component like this:
`export class Home extends React.Component { }`

Then, inside those curly braces, a React component should always have a `render()` method, like so:
`render() {
        return(
            <div className="home-container">
                <div className="home-left">
                       <Sidebar />
                </div>
               <div className="home-right"></div>
         </div>
        )
    }`

(shame my comment isn't formatted very well, but you get the idea :p )

This also removed the empty `<>`

Reply:
I realise it's probs easier with an image. Something like this:

![Screen Shot 2020-04-17 at 13 36 38](https://user-images.githubusercontent.com/39953553/79570085-b6eff500-80b0-11ea-8b00-a4df219089ca.png)

Reply:
Is there a reason you would suggest class components as opposed to functional components with hooks?
React seem to recommend the latter https://reactjs.org/docs/hooks-intro.html

Reply:
Primarily it is familiarity. If you guys would rather use functional and hooks, that's fine by me 👍

Reply:
I've been taught against using class components as  with hooks they no longer provide any extra functionality. All the React mentors at newcastle were against class components and theres been talk of getting functional components in the training doc

Reply:
Yeah that's fair. Proceed